client/src/view/info-pages.tsx

```diff
@@ -54,25 +30,20 @@ export const INFO_PAGES: JSX.Element[] = [
     (
         <div>
             <p>
-                As the story goes, <strong>galapa</strong> means something like
-                "family" in the local dialect and a <strong>hexalot</strong>,
-                is a unique beehive pattern of surface spots where a *gotchi* lives.
+                Every "hexalot" is a place with a completely unique pattern of spots
+                and it is forever connected to its neighbors.
             </p>
             <p>
-                Every *hexalot* is a place with a completely unique pattern
-                and it is forever connected to its neighbors.
+                A "gotchi" emerges from a tower on the middle spot of a hexalot,
```

Comment:
Yes, but what _is_ a "gotchi"? You don't explain that

Reply:
A "gotchi" is a virtual animal which is born from an egg in the tower in the middle of a hexalot, evolving first body shape and then muscle coordination.

Reply:
Excellent, though i think you can drop "in the tower"

Reply:
Ok.

Reply:
A "gotchi" is a virtual animal which is born and reborn from an egg at the middle of a hexalot.